.concourse/pipeline.yaml.gomplate

```diff
@@ -1119,6 +1119,10 @@ jobs:
     - put: semver.gke-cluster
       params: {bump: patch}
   - task: upgrade
+    params:
+      GKE_KEY: '((gke-suse-cap-json))'
+      GKE_PROJECT: '{{ if has $ "gke_project" }}{{ $.gke_project }}{{ else }}suse-225215{{ end }}'
+      GKE_ZONE: '{{ if has $ "gke_zone" }}{{ $.gke_zone }}{{ else }}europe-west3-c{{ end }}'
```

Comment:
```suggestion
      GKE_ZONE: {{ default "europe-west3-c" $.gke_zone | quote }}
```

Reply:
:warning: gomplate __does not have__ a `default` function. Using this form fails when the value is not set.